contents/docs/libraries/flutter/_snippets/identify.mdx

```diff
@@ -0,0 +1,23 @@
+When you start tracking events with PostHog, each user gets an anonymous ID that is used to identify them in the system.
+In order to link this anonymous user with someone from your database, use the `identify` call. 
+
+Identify lets you add metadata to your users so you can easily identify who they are in PostHog, as well as do things 
+like segment users by these properties.
+
+An identify call requires:
+
+* `userId` which uniquely identifies your user in your database
+* `userProperties` with a dictionary of key:value pairs
```

Comment:
Missing `userPropertiesSetOnce`

Reply:
Its missing from iOS and android too. Will add these to all of them in a separate PR